docker/Dockerfile_txsim

```diff
@@ -0,0 +1,48 @@
+# stage 1 Generate celestia-appd Binary
+FROM docker.io/golang:1.20.6-alpine3.17 as builder
+# hadolint ignore=DL3018
+RUN apk update && apk add --no-cache \
+    gcc \
+    git \
+    make \
+    musl-dev
+COPY . /celestia-app
+WORKDIR /celestia-app
+# we need the appd build as we might want to create an account
+# internally for txsimulation
+RUN make build && make txsim-build
+
+# stage 2
+FROM docker.io/alpine:3.18.2
+
+# Read here why UID 10001: https://github.com/hexops/dockerfile/blob/main/README.md#do-not-use-a-uid-below-10000
```

Comment:
This comment is helpful because it includes a link but it forces the reader to context switch to reading an article. How about:

```suggestion
# Use UID 10,001 because UIDs below 10,000 are a security risk.
# Ref: https://github.com/hexops/dockerfile/blob/main/README.md#do-not-use-a-uid-below-10000
```